### PR TITLE
6.5.0: Add sensuctl configure option for env vars

### LINKED ISSUE #3340: sensuctl configure and environment variables
### Description
In 6.5.0, the global flags for sensuctl configure can be set as environment variables:
```
      --api-key string             API key to use for authentication
      --api-url string             host URL of Sensu installation
      --cache-dir string           path to directory containing cache & temporary files (default "/Users/hillaryfraley/Library/Caches/sensu/sensuctl")
      --config-dir string          path to directory containing configuration files (default "/Users/hillaryfraley/.config/sensu/sensuctl")
      --insecure-skip-tls-verify   skip TLS certificate verification (not recommended!)
      --namespace string           namespace in which we perform actions (default "default")
      --timeout duration           timeout when communicating with sensu backend (default 15s)
      --trusted-ca-file string     TLS CA certificate bundle in PEM format
```

Also, environment variables can be set in the same command with sensuctl configure (does not require sensuctl env).

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
https://docs.sensu.io/sensu-go/latest/sensuctl/environment-variables/

### Corrections needed
- Document new sensuctl configure global flags as environment variable options
- Explain how to set env vars with sensuctl configure

### Additional context
Issue: https://github.com/sensu/sensu-enterprise-go/issues/1097
PR: https://github.com/sensu/sensu-enterprise-go/pull/1767

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds sensuctl configure option for defining environment variables in https://docs.sensu.io/sensu-go/latest/sensuctl/environment-variables/

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/3340

### CODE REVIEW

File: content/sensu-go/6.5/sensuctl/environment-variables.md [hunk 1]
```diff
@@ -12,20 +11,72 @@ menu:
     parent: sensuctl
 ---
 
-Sensuctl includes the `sensuctl env` command to help export and set environment variables on your systems.
+Sensu allows you to set sensuctl environment variables for a [single sensuctl command][1], with [sensuctl configure][2], or with [sensuctl env][3].
```
Comment (hillaryfraley) [inline content/sensu-go/6.5/sensuctl/environment-variables.md]: I think we should acknowledge that you can do all this configuration with flags instead, but I'm not sure why a customer would use one or the other. Is it just a matter of preference? Or are there functionality differences between flags and env vars that we need to point out here?

Comment (hillaryfraley) [inline content/sensu-go/6.5/sensuctl/environment-variables.md]: Also, the first thing I wondered when trying to sort this out is why the `sensuctl env` env var options are different from the env var options for the other two methods. Is there a reason we can provide to help prevent confusion?

File: content/sensu-go/6.5/sensuctl/environment-variables.md [hunk 2]
```diff
@@ -12,20 +11,72 @@ menu:
...
+{{< code shell >}}
+SENSU_OIDC=true SENSU_NON_INTERACTIVE=true SENSU_FORMAT=yaml SENSU_PORT=7999 SENSU_TIMEOUT=49s SENSU_URL=http://192.168.7.217:8080 sensuctl configure
+{{< /code >}}
+
+Setting environment variables in a sensuctl configure command allows you to inject the desired environment variables in an automation script, quickly and transparently configuring sensuctl.
+
+Environment variables set with `sensuctl configure` are persistent.
+
+## Set environment variables with sensuctl env
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.5/sensuctl/environment-variables.md:71]: I think the use cases for single-command and configure methods are clear (or at least have some explanation). But it seems like we need to add some explanation about when and why to use `sensuctl env`.

Comment (fguimond) [reply content/sensu-go/6.5/sensuctl/environment-variables.md:71]: I personally have used `sensuctl env` to export the environment in my shell to the use them with `curl` or other scripts. I'm not sure what the use cases were when this was written. AFAIK it's used to export the environment from sensuctl, not set it.

File: content/sensu-go/6.5/sensuctl/environment-variables.md (hunk 1 above)
Comment (fguimond) [reply content/sensu-go/6.5/sensuctl/environment-variables.md]: Functionality is the same as using the flags. As for the why question I can see these scenarios:
* customer has two different environments - having one shell with a set of variables and another one with another set prevents you from having to run sensuctl configure every time you go from one to the other
* security-wise if someone intercepts your `sencuctl configure` command, if you're using flags the sensitive information is visible whereas it's not if using environment
* environment is also easier to pass into a container creation script

I'm sure @calebhailey is able to provide additional information on the operational side of Sensu that makes this useful.

Comment (fguimond) [reply content/sensu-go/6.5/sensuctl/environment-variables.md]: The environment variables are directly taken from the equivalent flags. I don't know how the variables were selected when `sensuctl env` was created. Again I'm sure @calebhailey or @portertech have more insights.

**Comment by hillaryfraley** [comment]
Thank you, @fguimond ! I incorporated your clarifications.

**Review by fguimond — APPROVED** [review]

**Pull request merged**